lib/resources/dstu2/medication_statement.yaml

```diff
@@ -17,85 +17,90 @@ fields:
+  example: '{"patient": {"reference": "Patient/5366327"}}'
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: >
+                 A code representing the patient or other source's judgment about the state of the medication
+                 used that this statement is about.
+  example: '{"status": "active"}'
+
+- name: effectivePeriod
+  required: 'No'
+  cardinality: 0..1
+  type: Period
+  description: The interval of time during which it is being asserted that the patient was taking the medication.
+  note: If effectivePeriod and dosage.timing.repeat.boundsPeriod are both populated they need to be the same value.
   example: |
-               {"contained": [
-                 {"resourceType": "Medication",
-                    "id": "2750067",
-                    "code": {
-                      "coding": [
-                        {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                         "code": "1191",
-                         "display": "Aspirin",
-                         "userSelected": false}],
-                      "text": "aspirin"}}]}
-  url: http://hl7.org/fhir/DSTU2/domainresource-definitions.html#DomainResource.contained
-  note: It should contain medication resource with medication.code
-  children:
+            {
+                "effectivePeriod": {
+                    "start": "2015-05-05T14:00:00.000Z",
+                    "end": "2015-05-06T01:00:00.000Z"
+                }
+            }
+  url: http://hl7.org/fhir/DSTU2/medicationstatement-definitions.html#MedicationStatement.effective_x_
+  action: create
 
-  - name: code
-    required: 'No'
-    cardinality: 0..1
-    type: CodeableConcept
-    description:  Codes that identify this medication.
-    example: |
-               {"code": {
-                 "coding": [
-                   {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
-                    "code": "1191",
-                    "display": "Aspirin",
-                    "userSelected": false}],
-                 "text": "aspirin"}}
-    url: http://hl7.org/fhir/DSTU2/medication-definitions.html#Medication.code
-    note: This code is only belongs where the resource is a Medication.
-    binding:
-      description: A code that defines the type of medication.
-      terminology:
-      - display: RxNorm
-        system: http://www.nlm.nih.gov/research/umls/rxnorm
+- name: note
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  description: Provides extra information about the medication statement that is not conveyed by the other attributes.
+  example: '{"note": "do not take with alcohol"}'
+  action: create
+
+- name: medication[x]
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept | contained Reference (Medication)
+  description: Identifies the medication being administered.
+  note:
+  - update: This must be the current medication. It cannot be modified.
+  example: |
+             {
```

Comment:
Can you update your examples so that the formatting is consistent with the existing resources?
- Opening grouper on the initial line
- Indent 2 spaces
- Closing groupers compacted onto a single line

Main reason, other than consistency, is to try to keep the definition table from scrolling